modules/react-components/src/components/transfer/transfer-list.tsx

```diff
@@ -101,7 +101,7 @@ export const TransferList: FunctionComponent<TransferListPropsInterface> = (
                             subtitle={ [
                                 emptyPlaceholderContent
                                     ? emptyPlaceholderContent
-                                    : t("devPortal:components.transferList.list.emptyPlaceholders.default")
+                                    : t("console:manage.features.transferList.list.emptyPlaceholders.default")
```

Comment:
Add a TODO and an issue to remove the usage of translation namespaces inside `React Components` module.

Reply:
Fixed with commit 76d6c86. This issue will track the mentioned issue. https://github.com/wso2/product-is/issues/10693